### ISSUE TITLE
Compare two loss functions

### ISSUE THREAD

**Issue by uchuuronin** [riakashyap/646_Project#61]
Create a small script to compare the finetuned model w loss function approach by E2Rank vs Pairwise loss func to determine better approach
